### PR TITLE
TELCODOCS-1932: Implement post-GA SME feedback

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.16, 4.17

Issue: https://issues.redhat.com/browse/TELCODOCS-1932

Link to docs preview:

- https://78439--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.html
- https://78439--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.html
- https://78439--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-generate-seed.html
- https://78439--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-partition.html
- https://78439--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/ztp-image-based-upgrade.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cnf-image-based-upgrade-shared-container-partition.adoc [hunk 1]
```diff
@@ -30,11 +30,11 @@ spec:
       version: 3.2.0
     storage:
       disks:
-        - device: /dev/disk/by-id/wwn-<root_disk> <1>
+        - device: /dev/disk/by-path
```
[the remark this answers is not in the document]
Comment (Missxiaoguo) [reply modules/cnf-image-based-upgrade-shared-container-partition.adoc:33]: yes, you need to provide a device path `/dev/disk/by-path/pci-<root_disk>` and replace `/dev/disk/by-id/wwn-...` with an example device path in all `ignitionConfigOverride` and example output.

https://docs.openshift.com/container-platform/4.16/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-partition.html#ztp-image-based-upgrade-shared-container-partition_shared-container-partition

File: modules/ztp-image-based-upgrade-shared-container-partition.adoc [hunk 2]
```diff
@@ -26,7 +26,7 @@ variant: fcos
 version: 1.3.0
 storage:
   disks:
-  - device: /dev/disk/by-id/wwn-<root_disk> <1>
+  - device: /dev/disk/by-path/pci-0000:00:17.0-ata-1.0 <1>
```
Comment (Missxiaoguo) [inline modules/ztp-image-based-upgrade-shared-container-partition.adoc:29]:
```suggestion
  - device: /dev/disk/by-path/pci-<root_disk> <1>
```

File: modules/cnf-image-based-upgrade-seed-image-guide.adoc [hunk 3]
```diff
@@ -4,7 +4,7 @@
 [id="cnf-image-based-upgrade-seed-image-guide_{context}"]
 = Seed image guidelines
 
-The seed image targets a set of {sno} clusters with similar configuration.
+The seed image targets a set of {sno} clusters with the same hardware and similar configuration.
```
Comment (mpmaruthu) [inline modules/cnf-image-based-upgrade-seed-image-guide.adoc:7]: I would like to update customer document with  below system configuration.

`The CPU topology must align with the seed SNO.
Same number of cores.
Same performance tuning (i.e., reserved CPUs).`

Ref:- https://github.com/openshift-kni/lifecycle-agent/blob/main/docs/seed-image-generation.md#seed-sno-pre-requisites

File: modules/cnf-image-based-upgrade-seed-image-config.adoc [hunk 4]
```diff
@@ -5,7 +5,7 @@
 [id="cnf-image-based-upgrade-seed-image-config_{context}"]
 = Seed image configuration
 
-The seed image targets a set of {sno} clusters with similar configuration.
+The seed image targets a set of {sno} clusters with the same hardware and similar configuration.
```
Comment (mpmaruthu) [inline modules/cnf-image-based-upgrade-seed-image-config.adoc:8]: Ref:- https://github.com/openshift-kni/lifecycle-agent/blob/main/docs/seed-image-generation.md#seed-sno-pre-requisites

I would like to update customer document with below system configuration.

`The CPU topology must align with the seed SNO. Same number of cores. Same performance tuning (i.e., reserved CPUs).`

**Comment by mpmaruthu** [comment]
Telco RAN QE: Looks good to me.
/Peri M

**Review by mpmaruthu — APPROVED** [review]

File: modules/cnf-image-based-upgrade-backup-guide.adoc [hunk 5]
```diff
@@ -20,7 +20,7 @@ The following resources must be excluded from the backup:
 
 There are two local storage implementations for {sno}:
 
-Local Storage Operator (LSO):: The {lcao} backs up and restores the required artifacts, including `LocalVolumes` and their associated `StorageClasses`. You must exclude the `persistentVolumes` resource in the application `Backup` CR.
+Local Storage Operator (LSO):: The {lcao} automatically backs up and restores the required artifacts, including `LocalVolumes` and their associated `StorageClasses`. You must exclude the `persistentVolumes` resource in the application `Backup` CR.
```
Comment (abrennan89) [inline modules/cnf-image-based-upgrade-backup-guide.adoc:23]:
```suggestion
Local Storage Operator (LSO):: The {lcao} automatically backs up and restores the required artifacts, including `LocalVolume` resources and their associated `StorageClass` resources. You must exclude the `PersistentVolumes` resource in the application `Backup` CR.
```
I think "`LocalVolume` resources" and "`StorageClass` resources" should be formatted this way, per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references:
image

Also, I think `PersistentVolumes` should be capitalized, per other references, for example: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reserving-a-persistentvolume and https://docs.openshift.com/container-platform/4.16/storage/persistent_storage/persistent_storage_local/persistent-storage-local.html#local-create-cr-manual_persistent-storage-local

File: modules/cnf-image-based-upgrade-seed-image-config.adoc [hunk 6]
```diff
@@ -52,6 +52,7 @@ a|Local volumes
 |No
 |====
 . Dual-stack networking is not supported in this release.
+. If the seed cluster is installed in a disconnected environment, the target clusters must be installed in a disconnected environment too.
```
Comment (abrennan89) [inline modules/cnf-image-based-upgrade-seed-image-config.adoc:55]:
```suggestion
. If the seed cluster is installed in a disconnected environment, the target clusters must also be installed in a disconnected environment.
```
Suggested wording update

**Review by abrennan89 — COMMENTED** [review]
A couple of comments / suggestions

File: modules/cnf-image-based-upgrade-backup-guide.adoc (hunk 5 above)
Comment (amolnar-rh) [reply modules/cnf-image-based-upgrade-backup-guide.adoc:23]: Re capitalization:
I'm not sure that in the Backup/Restore CRs you need to capitalize the resource names. Here's where the examples are from: https://github.com/openshift-kni/lifecycle-agent/blob/main/docs/backuprestore-with-oadp.md#application-backup-and-restore-crs

@Missxiaoguo Could you please take a look and confirm whether we need to capitalize in this case?

Comment (Missxiaoguo) [reply modules/cnf-image-based-upgrade-backup-guide.adoc]: The resources in the Backup/Restore CR could be either capitalized or not. For example,  all of these should work.  
```
  names:
    kind: LocalVolume
    plural: localvolumes
    singular: localvolume
```

File: modules/cnf-image-based-upgrade-backup-guide.adoc [hunk 7]
```diff
@@ -20,7 +20,7 @@ The following resources must be excluded from the backup:
 
 There are two local storage implementations for {sno}:
 
-Local Storage Operator (LSO):: The {lcao} backs up and restores the required artifacts, including `LocalVolumes` and their associated `StorageClasses`. You must exclude the `persistentVolumes` resource in the application `Backup` CR.
+Local Storage Operator (LSO):: The {lcao} automatically backs up and restores the required artifacts, including `localvolume` resources and their associated `StorageClass` resources. You must exclude the `persistentVolumes` resource in the application `Backup` CR.
```
Comment (Missxiaoguo) [inline modules/cnf-image-based-upgrade-backup-guide.adoc:23]: I think we want to change `persistentVolumes` to `persistentvolumes` in the backup CR example 
```suggestion
Local Storage Operator (LSO):: The {lcao} automatically backs up and restores the required artifacts, including `localvolume` resources and their associated `StorageClass` resources. You must exclude the `persistentvolumes` resource in the application `Backup` CR.
```

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**